mediation_fees.rst

```diff
@@ -107,115 +107,124 @@ easier to let them define a *per-hop* proportional mediation fee (called
 Fee calculation
 ===============
 
-There are two fundamental formula to relate :math:`a`, :math:`b` and :math:`c`.
-
-1. :math:`a = c + aq + f + i(-a)`
-
-2. :math:`c = b + bq + f + i(b)`
-
-The other fundamental relations are:
-
-- :math:`a - {fee}_{in} = c`
-- :math:`c - {fee}_{out} = b`
-
-The imbalance fee :math:`i(x)` is defined as follows, where :math:`t` is the channel capacity, :math:`x` is the transferred amount and :math:`\mathit{IP}(\mathit{capacity})` is the imbalance penalty function.
+The mediation fees for a payment amount :math:`x` for a single channel are calculated as
 
 .. math::
 
-    i(x) = \mathit{IP}(t + x) - \mathit{IP}(t)
-
-In (1) we pass the negative amount to :math:`i` because the incoming channel's balance is decreased by the transfer, while it is increased for outgoing channel in (2) .
-
-
-.. note::
-
-    These equations only have symbolic solutions when no imbalance fees are used. With imbalance fees only approximate solutions are presented below. This means that forward and backwards fee calculations can differ slightly.
+   \mathit{fee}_{\mathit{channel}}(x) := \mathit{flat} + q|x| + i(x)
```

Comment:
Why do we need to take the absolute value here? Payments should always be positive, right?

Reply:
We can either have i_forward(x) and i_backward(x) for the imbalance_fees or treat the amount as positive/negative for the incoming/outgoing channel. I choose the latter approach, which requires the addition of the absolute.

Reply:
Makes sense. We can leave it like it is.